scan_topic: "/scan"
queue_size: 10
p_gain: 1
x_dot: 0.5